composer/loggers/mlflow_logger.py

```diff
@@ -118,6 +120,11 @@ class MLFlowLogger(LoggerDestination):
         logging_buffer_seconds (int, optional): The amount of time, in seconds, that MLflow
             waits before sending logs to the MLflow tracking server. Metrics/params/tags logged
             within this buffer time will be grouped in batches before being sent to the backend.
+        log_duplicated_metric_every_n_steps (int, optional): The number of steps to wait before
+            logging the duplicated metric value. Duplicated metric value means the new step has the
+            same value as the previous step. (default: ``100``)
+        log_duplicated_metric_every_n_millis (int, optional): The number of milliseconds to wait
+            before logging the duplicated metric value. (default: ``600000``)
```

Comment:
honestly this is probably fine to remove, I can't think of a good use case for this and it'll simplify things

Reply:
sg!